Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageDetails.uxml

```diff
@@ -0,0 +1,28 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" editor-extension-mode="False">
+    <Style src="project://database/Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageWizard.uss?fileID=7433441132597879392&amp;guid=0c061ff59c04373489742356252f97ea&amp;type=3#PackageWizard" />
+    <ui:VisualElement name="PackageDetails">
+        <ui:Label text="Package Details" name="PackageDetailsTitle" class="section-title" />
+        <ui:TextField picking-mode="Ignore" label="Package Name" value="filler text" name="package-name" style="border-left-width: 6px; border-right-width: 6px; border-top-width: 6px; border-bottom-width: 6px; margin-left: 0; margin-right: 0; margin-top: 0; margin-bottom: 0; padding-left: 0; padding-right: 0; padding-top: 0; padding-bottom: 0;" />
```

Comment:
Not urgent, but we should be aiming to move and in-line styles into USS classes, as that should help reduce the clutter of the UXML as well as make things easier to edit

Reply:
Might as well get in the habit of it now!

Reply:
Yeah let's clean this up, It's excellent habit to get into. Inline sucks

Reply:
I've updated the uxmls! I left one inline style in the PackageCreation.uxml for a button that will be used later.